quodlibet/po/fr.po

```diff
@@ -523,9 +513,8 @@ msgid "Internet Radio"
 msgstr "Radio Internet"
 
 #: ../quodlibet/browsers/iradio.py:252
-#, fuzzy
 msgid "Downloading station list"
-msgstr "Télécharger les fichiers"
+msgstr "Télécharger la liste de station"
```

Comment:
@trebmuh - People would work out what it meant, but it'd never be written like that idiomatically.